src/junit_xml.jl

```diff
@@ -28,7 +28,7 @@ end
 mutable struct JUnitTestCase  # TestItem run
     const name::String
     counts::JUnitCounts
-    stats::PerfStats
+    stats::Union{PerfStats, Nothing} # Additional stats not available from the testset
     error_message::Union{String,Nothing} # Additional message to include in `<error>`
     logs::Union{Vector{UInt8},Nothing} # Captured logs from a `@testitem`
```

Comment:
do logs need to be a vector of `UInt8`? Or is a string okay here? The way my logs are captured they are of type `String`

Reply:
Actually, I take that back. I end up writing everything to a buffer when it gets formatted, so it is a `Vector{UInt8}`